src/encoded/static/components/biosample.js

```diff
@@ -34,9 +35,10 @@ class BiosampleComponent extends React.Component {
         // Build the text of the synchronization string
         let synchText;
         if (context.synchronization) {
+            const synchronizationTime = Measurement.format(context.post_synchronization_time, context.post_synchronization_time_units);
             synchText = context.synchronization +
                 (context.post_synchronization_time ?
-                    ` + ${context.post_synchronization_time}${context.post_synchronization_time_units ? ` ${context.post_synchronization_time_units}` : ''}`
+                    `${synchronizationTime}`
```

Comment:
Don’t need the string template; just `{synchronizationTime}` would work.

Reply:
Good catch.